inst/rmarkdown/templates/wellcomeor/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,196 @@
+---
+title: Wellcome Open Research Article Template
+titlenote: The title should be detailed enough for someone to know whether the article would be of interest to them, but also concise. Please ensure the broadness and claims within the title are appropriate to the content of the article itself.
+author:
+  - name: Author Name1
+    affilliation: 1
+  - name: Author Name2
+    affilliation: 2
+address:
+  - code: 1
+    address: Address1
+  - code: 2
+    address: Address2
+abstract: |
+  Abstracts should be up to 300 words and provide a succinct summary of the article. Although the abstract should explain why the article might be interesting, care should be taken not to inappropriately over-emphasise the importance of the work described in the article. Citations should not be used in the abstract, and the use of abbreviations should be minimized. If you are writing a Research or Systematic Review article, please structure your abstract into Background, Methods, Results, and Conclusions.
+keywords: |
+  Please list up to eight keywords to help readers interested in your article find it more easily.
+
+date: "`r Sys.Date()`"
+bibliography: sample.bib
+biblio-style: "unsrtnat"
+link-citations: TRUE
+colorlinks: TRUE
```

Comment:
This is what creates your link coloring. You should not set it if you don't want link color.

Reply:
I was having issues with this not doing what I expected, but I think that's been addressed in a recent commit

Reply:
There could be another way - to follow pandoc template directly. 
https://github.com/jgm/pandoc/blob/master/data/templates/default.latex

I'll propose you some changes toward that.